Framework/HistogramData/test/ScalableTest.h

```diff
@@ -60,6 +78,14 @@ class ScalableTest : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(lhs[1], 0.6, 1e-14);
   }
 
+  void test_divide_reverse_order() {
```

Comment:
I am conflicted about whether or not a divide in reverse order should be allowed. I like the idea but not sure if it really makes sense to allow this. What do you think @OwenArnold ?